source/appModules/poedit.py

```diff
@@ -79,42 +91,61 @@ def _get__dataViewControlId(self) -> int | None:
 			return None
 		return dataView.windowControlID
 
+	_sidebarControlId: int | None
+	"""Type definition for auto prop '_get__sidebarControlId'"""
+
+	def _get__sidebarControlId(self) -> int | None:
+		dataViewControlId = self._dataViewControlId
+		splitterControlID = dataViewControlId + _WindowControlIdOffsetFromDataView.MAIN_SPLITTER_IDENTIFIER
+		fg = api.getForegroundObject()
+		splitterHwnd = windowUtils.findDescendantWindow(fg.windowHandle, controlID=splitterControlID)
+		print(f"splitter: {winUser.getClassName(splitterHwnd)}")
+		sidebarHwnd = winUser.getWindow(splitterHwnd, winUser.GW_HWNDNEXT)
+		while sidebarHwnd and not ctypes.windll.user32.IsWindowVisible(sidebarHwnd):
+			print(f"skipping non-sidebar: {winUser.getClassName(sidebarHwnd)}")
+			sidebarHwnd = winUser.getWindow(sidebarHwnd, winUser.GW_HWNDNEXT)
+		print(f"sidebar? {winUser.getClassName(sidebarHwnd)}")

```

Comment:
I think the content of this message needs updating if it remains as a log message.
I'm going to suggest removal here unless a more descriptive message is used

```suggestion
```